CONTRIBUTING.md

```diff
@@ -36,9 +36,13 @@ In order to run most of the available tests, one would need to run:
 
 `./tools/run_tests/run_tests.py`
 
-If you want to run all the possible tests for any of the languages {c, c++, node, php, python}, do this:
+If you want to run tests for any of the languages {all, c, c++, csharp, node, objc, php, python, ruby}, do this:
```

Comment:
"all" is only supported for -c gcov (only as a workaround and that change is not even merged yet). Please remove "all" from the list